config/defaults.js

```diff
@@ -1,3 +1,3 @@
 module.exports = {
-  apiPrefix: "http://localhost:3000/api/v1"
+  apiPrefix: "http://localhost:5000/api/v1"
```

Comment:
ah sorry i missed this comment. i would guess heroku local web is just app agnostic so it has its own default port. but we can [give it a specific port](https://devcenter.heroku.com/articles/heroku-local) to run on if we want, eg. `heroku local web -p 3000`. this would keep our dev environments more agnostic to which way you want to run the server. 

does that sound good?